### ISSUE TITLE
Monokle throws error on Ubuntu Studio when installed via Snap package

### ISSUE THREAD

**Issue by pcgeek86** [kubeshop/monokle#2766]
**Describe the bug**

I installed Monokle on Ubuntu Studio, using the Snap package, and attempted to run it. I then received the error below.

```
A JavaScript error occurred in the main process
Uncaught Exception:
Error [ERR_REQUIRE_ESM]: require() of ES Module /snap/monokle/2/resources/app.asar/node_modules/default-shell/index.js from /snap/monokle/2/resources/app.asar/build/src/utils/env.js not supported.
Instead change the require of index.js in /snap/monokle/2/resources/app.asar/build/src/utils/env.js to a dynamic import() which is available in all CommonJS modules.
    at c._load (node:electron/js2c/asar_bundle:5:13343)
    at Object.<anonymous> (/snap/monokle/2/resources/app.asar/build/src/utils/env.js:10:25)
    at c._load (node:electron/js2c/asar_bundle:5:13343)
    at Object.<anonymous> (/snap/monokle/2/resources/app.asar/build/src/utils/path.js:4:15)
    at c._load (node:electron/js2c/asar_bundle:5:13343)
    at Object.<anonymous> (/snap/monokle/2/resources/app.asar/build/electron/app/index.js:13:16)
    at c._load (node:electron/js2c/asar_bundle:5:13343)
    at Object.<anonymous> (/snap/monokle/2/resources/app.asar/build/electron/main.js:26:1)
    at c._load (node:electron/js2c/asar_bundle:5:13343)
    at Object.<anonymous> (node:electron/js2c/browser_init:185:3104)
    at ./lib/browser/init.ts (node:electron/js2c/browser_init:185:3308)
    at __webpack_require__ (node:electron/js2c/browser_init:1:128)
    at node:electron/js2c/browser_init:1:1200
    at node:electron/js2c/browser_init:1:1267
    at c._load (node:electron/js2c/asar_bundle:5:13343)

(Monokle:2852): Gtk-WARNING **: 12:12:52.407: Theme parsing error: gtk.css:3536:25: 'font-feature-settings' is not a valid property name
Gtk-Message: 12:12:52.421: Failed to load module "colorreload-gtk-module"
Gtk-Message: 12:12:52.421: Failed to load module "window-decorations-gtk-module"
[2852:1118/121252.448314:ERROR:object_proxy.cc(623)] Failed to call method: org.freedesktop.DBus.ListActivatableNames: object_path= /org/freedesktop/DBus: org.freedesktop.DBus.Error.AccessDenied: An AppArmor policy prevents this sender from sending this message to this recipient; type="method_call", sender=":1.93" (uid=1000 pid=2852 comm="/snap/monokle/2/Monokle --no-sandbox --enable-cras" label="snap.monokle.monokle (enforce)") interface="org.freedesktop.DBus" member="ListActivatableNames" error name="(unset)" requested_reply="0" destination="org.freedesktop.DBus" (bus)
[2922:1118/121252.560463:ERROR:viz_main_impl.cc(186)] Exiting GPU process due to errors during initialization
[2938:1118/121252.602204:ERROR:gpu_memory_buffer_support_x11.cc(44)] dri3 extension not supported.
```

**To Reproduce**
1. Install Ubuntu Studio
2. Install Monokle Snap package
3. Attempt to run Monokle

**Expected behavior**
Monokle runs without errors.

**Screenshots**

![image](https://user-images.githubusercontent.com/466713/202784961-adf90047-1e65-436f-a70f-e00f94d86b12.png)

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Ubuntu Studio
 - Version [e.g. 22] 22.04 LTS

**Comment by chargio** [comment]
We believe this bug has been fixed with the recent changes  in 2.0, we are evaluating if we can port it to 1.13.
(1.13 will be December, 2.0 late January). We will have to check back once the fix is released

**Comment by chargio** [comment]
Closing the issue because Snap is not released yet
